packages/peregrine/lib/talons/CheckoutPage/PaymentInformation/usePaymentInformation.js

```diff
@@ -1,13 +1,16 @@
-import { useCallback, useEffect, useState } from 'react';
-import { useMutation, useQuery } from '@apollo/react-hooks';
+import { useCallback, useState, useEffect } from 'react';
+import { useQuery, useApolloClient, useMutation } from '@apollo/react-hooks';
 
 import { useAppContext } from '../../../context/app';
 import { useCartContext } from '../../../context/cart';
+import CheckoutError from '../CheckoutError';
 import { CHECKOUT_STEP } from '../useCheckoutPage';
 
 /**
  *
  * @param {Function} props.onSave callback to be called when user clicks review order button
+ * @param {Object} props.checkoutError an instance of the `CheckoutError` error that has been generated using the error from the place order mutation
+ * @param {DocumentNode} props.queries.getPaymentDetailsQuery query to fetch selected payment method and payment nonce details
```

Comment:
Replace this with the nonce query doc. the other query is already included in the docs on line 16.